pkg/controller/template/template_controller.go

```diff
@@ -102,27 +111,29 @@ func New(
 		DeleteFunc: ctrl.deleteSecret,
 	})
 
+	featureInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    ctrl.addFeature,
+		UpdateFunc: ctrl.updateFeature,
+		DeleteFunc: ctrl.deleteFeature,
+	})
+
 	ctrl.syncHandler = ctrl.syncControllerConfig
 	ctrl.enqueueControllerConfig = ctrl.enqueue
 
 	ctrl.ccLister = ccInformer.Lister()
 	ctrl.mcLister = mcInformer.Lister()
+	ctrl.featLister = featureInformer.Lister()
 	ctrl.ccListerSynced = ccInformer.Informer().HasSynced
 	ctrl.mcListerSynced = mcInformer.Informer().HasSynced
 	ctrl.secretsInformerSynced = secretsInformer.Informer().HasSynced
+	ctrl.featListerSynced = featureInformer.Informer().HasSynced
 
 	return ctrl
 }
 
 func (ctrl *Controller) filterSecret(secret *corev1.Secret) {
 	if secret.Name == "pull-secret" {
-		cfg, err := ctrl.ccLister.Get(ctrlcommon.ControllerConfigName)
-		if err != nil {
-			utilruntime.HandleError(fmt.Errorf("couldn't get ControllerConfig on secret callback %#v", err))
-			return
-		}
-		glog.V(4).Infof("Re-syncing ControllerConfig %s due to secret change", cfg.Name)
```

Comment:
I keep coming back to removing this secret change logging... Can you leave the log line and maybe with a reword for it to make sense since there's another log line in the func called ?

Reply:
As an aside: @sinnykumari should this logging really be v4? on secret change maybe we want it to be more visible? is there an event for this somewhere?

Reply:
yes, there is no confidential information we are logging here. So, it makes sense to log it without verbosity.

Reply:
Added the log line without verbosity